.github/workflows/components-testing.yml

```diff
@@ -0,0 +1,42 @@
+name: Components testing
+
+on:
+  push:
+    paths:
+      - 'package/design-system/**'
+
+jobs:
+  cypress-run:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v2
+
+      - uses: actions/setup-node@v2
+        with:
+          node-version: '14'
+
+      - name: Yarn cache directory
+        id: yarn-cache-dir-path
+        run: echo "::set-output name=dir::$(yarn cache dir)"
+
+      - name: Yarn cache
+        uses: actions/cache@v2
+        id: yarn-cache
+        with:
+          path: ${{ steps.yarn-cache-dir-path.outputs.dir }}
+          key: ${{ runner.os }}-yarn-${{ hashFiles('**/yarn.lock') }}
+          restore-keys: |
+            ${{ runner.os }}-yarn-
+
+      - name: Cypress Component Testing
+        run: |
+          yarn
+          npx cypress run-ct
+
+      - name: Cypress artifacts upload
+        uses: actions/upload-artifact@v2
+        if: ${{ always() }}
```

Comment:
could you explain this one please ? looks like if true.

Reply:
because if it fails, we do want the video to know why!

Reply:
great ! I will remember this one :)